cmd/main.go

```diff
@@ -645,6 +651,13 @@ func (a *ApiEnabler) Enable() {
 	a.log.Info("API is enabled")
 }
 
+func (a *ApiEnabler) WaitForEnabled() error {
```

Comment:
this function never return an error so maybe it doesn't need to return a value at all?